src/ZoneServer/World/Actors/Characters/CharacterProperties.cs

```diff
@@ -255,6 +255,28 @@ private void UpdateRecoveryTimes(Character character)
 		{
 			this.Invalidate(PropertyName.RHPTIME, PropertyName.RSPTIME);
 			Send.ZC_OBJECT_PROPERTY(this.Character, PropertyName.RHPTIME, PropertyName.RSPTIME);
+
+			if (character.IsSitting)
```

Comment:
I feel like this, too, would fit very well into a subscriber of a "player sat down" event. It's not really related to properties in any way, except that the resulting buffs might affect them.

Reply:
wait, you just told me that I shouldn't subscriber to its own event, now you are telling the opposite.
Or I got it wrong?